docs/framework/deployment/windows/7.md

```diff
@@ -0,0 +1,17 @@
+# Installing the .NET Framework on Windows 7
+
+You may need the .NET Framework to run an application on Windows 7. You can use the following instructions to help you. The instructions depend on which version of the .NET Framework you need.
+
+## .NET Framework 4.6.2
+
+You can install the [.NET Framework 4.6.2](https://go.microsoft.com/fwlink/?linkid=845529&source=dotnetdocs) or later on Windows 7. 
```

Comment:
You can actually install every version since 4, though we recommend that customers install the latest version supported on their OS. So perhaps this and subsequent topics should be revised to reflect this.